src/components/Entry.vue

```diff
@@ -70,15 +77,75 @@
         </div>
       </button>
     </div>
+    <div v-show="isExtraInputValidationRequired" class="flex flex-col pt-10 pb-6">
+      <div>
+        <p
+          class="w-1/2 text-xl lg:text-2xl xl:text-3xl mx-auto font-bold md:w-3/4 text-center"
+        >
+          Enter your birthdate
+        </p>
+      </div>
```

Comment:
shouldn't this also be in the group data JSON on firestore?

Reply:
Ideally, yes. But haven't figured out the correct format for the JSON object in cases where there are additional input parameters. Have to think about it.

Reply:
got it
so maybe next big task for auth layer could be to
- learn from this sign up page stint and make some strings configurable which are hardcoded as of now
- and in general try to make auth layer such that adding a new auth profile should be automated, and not manual. like there should and must be a general way to add a new authentication type